### DISCUSSION TITLE
Rage bar overlay

### DISCUSSION THREAD

**Discussion by mariorojosalas** [Lailloken/Exile-UI#220] [Ideas & Feedback]
It would be cool if you could add a rage bar above life and mana bars in the character

**Comment by Lailloken** [comment]
You can set up a clone-frame for that.
![image](https://user-images.githubusercontent.com/61888437/230644599-8c1719cc-9aa5-4b9a-a2d8-0d15a06e35b8.png)

Just need to make sure the source frame is large enough because the text sometimes shifts a bit to the left or right when your max/reserved mana changes.
